nncf/torch/knowledge_distillation/algo.py

```diff
@@ -0,0 +1,64 @@
+from copy import deepcopy
```

Comment:
Please, add a license here and another files.

Reply:
Let's resolve it in a separate PR. created task for it #819 
Sergei can do it in the PR with docs and e2e test.